lib/plugins/aws/deploy/lib/updateStack.js

```diff
@@ -6,6 +6,8 @@ const path = require('path');
 
 module.exports = {
   update() {
+    const templateUrl = `https://s3.amazonaws.com/${this.bucketName}/compiled-cloudformation-template.json`;
```

Comment:
hmmm not sure about that. We don't wanna mess up the bucket, and the name should be predictable/constructible in case we wanna access it programmatically later. What's your thoughts on this @pmuens ?